### ISSUE TITLE
pvacfuse error with null values for peptide_sequence

### ISSUE THREAD

**Issue by boyangzhao** [griffithlab/pVACtools#996]
### Installation Type

Standalone

### pVACtools Version / Docker Image

pvactools 4.0.0

### Python Version

3.8

### Describe the bug

It seems that pvacfuse would return a message "Fusion position marker '*' not found in fusion sequence", if the peptide_sequence are empty. And I don't find that the * always need to be there, there are peptide_sequence values with the aa sequences, but no *, or contains "." that all passes the run with pvacfuse. In some of my arriba runs, I do get empty values for peptide_sequence, although rare, but this would trigger the message by pvacfuse. Can the tool ignore these rows? Otherwise I need to separately apply a pre-filter to get rid of these rows before submitting to pvacfuse.

Below is an example, using the example data, but with peptide_sequence replaced as empty, that fully reproduces the msg.

### How to reproduce this bug

```Shell
pvacfuse run test.tsv Test HLA-A*02:01 MHCflurry test_out -e1 10
```

### Input files

#gene1	gene2	strand1(gene/fusion)	strand2(gene/fusion)	breakpoint1	breakpoint2	site1	site2	type	split_reads1	split_reads2	discordant_mates	coverage1	coverage2	confidence	reading_frame	tags	retained_protein_domains	closest_genomic_breakpoint1	closest_genomic_breakpoint2	gene_id1	gene_id2	transcript_id1	transcript_id2	direction1	direction2	filters	fusion_transcript	peptide_sequence	read_identifiers
PTEN	ENSG00000200891(21548),MED6P1(31892)	+/+	./+	chr10:87952259	chr10:88016243	CDS/splice-site	intergenic	deletion/read-through	163	146	6	3691	3002	high	out-of-frame	.	C2_domain_of_PTEN_tumour-suppressor_protein(100%),Dual_specificity_phosphatase__catalytic_domain(100%)|	.	.	ENSG00000171862	.	ENST00000371953	.	downstream	upstream	duplicates(384),low_entropy(4),mismappers(15),mismatches(6),multimappers(2)	ACCAGAGACAAAAAG___GGAGTAACTATTCCCAGTCAGAGGCGCTATGTGTATTATTATAGCTACCTGTTAAAGAATCATCTGGATTATAGACCAGTGGCACTGTTGTTTCACAAGATGATGTTTGAAACTATTCCAATGTTCAGTGGCGGAACTTGCA|TGGGCAGATGTATGCAAACCTATCCTAAAGTTCAAGGAAGCTGAGAGGCCGAAGAAAGAGGCTGACAAATCTAGcTTCTCAGAAAGAAACCTTTAACAGGGATTTAGGAACAGAAGCCATGTCTGGGTCTCAGACAGGGCAGGATGAGATG			K00193:38:H3MYFBBXX:4:1101:18904:31572,K00193:38:H3MYFBBXX:4:1101:20386:3197

### Log output

Executing MHC Class I predictions
Converting Fusion file to TSV
Fusion position marker '*' not found in fusion sequence. If running with AGFusion results, please rerun AGFusion using the `--middlestar` option.

**Comment by susannasiebert** [comment]
Thank you for this bug report. I made a PR to handle this case. It's a bit strange because in our Arriba test runs empty peptide sequences were denoted with a `.` instead of leaving that column empty. I wonder if they changed their output format. With this change we will handle both cases.

**Comment by boyangzhao** [comment]
Thanks - in my outputs, Arriba also uses the `.`, but in rare instances, it has an empty column instead.

**Comment by susannasiebert** [comment]
This has been fixed in release 4.0.1. Please give it a try and let me know if you're still experiencing issues.

**Comment by boyangzhao** [comment]
thanks it's working now
